modules/utxo-lib/src/noble_crypto.ts

```diff
@@ -0,0 +1,76 @@
+import * as createHash from 'create-hash';
+import * as necc from '@noble/secp256k1';
+import { MuSigFactory, MuSig } from '@brandonblack/musig';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore base_crypto is exported as a subPath export, ignoring since compiler complains about importing like this
+import * as base_crypto from '@brandonblack/musig/base_crypto';
+
+function toBigInt(b: Uint8Array | Buffer): bigint {
+  return Buffer.from(b).readBigUint64BE();
+}
+
+const crypto = {
+  ...base_crypto,
+  pointMultiplyUnsafe: (p: Uint8Array, a: Uint8Array, compress: boolean): Uint8Array | null => {
```

Comment:
```suggestion
  pointMultiplyUnsafe(p: Uint8Array, a: Uint8Array, compress: boolean): Uint8Array | null {
```

for the other methods too